src/backend/ingestion/re-ingest.lambda.ts

```diff
@@ -1,4 +1,4 @@
-import { GetObjectCommand } from '@aws-sdk/client-s3';
+import { _Object, GetObjectCommand } from '@aws-sdk/client-s3';
```

Comment:
This looks unused???
```suggestion
import { GetObjectCommand } from '@aws-sdk/client-s3';
```